src/bindings/rust/elektra/src/kdb.rs

```diff
@@ -26,8 +26,10 @@ impl Drop for KDB {
 impl KDB {
     /// Opens the session with the Key database.
     pub fn open<'a>() -> Result<Self, KDBError<'a>> {
+        // FIXME: binding
```

Comment:
@PhilippGackstatter Should I keep anything in mind here, you can I just add the new KeySet parameter? Also would using `std::option` be a good idea, since the KeySet can be NULL in C?

Reply:
I think we can just add it. We could do the signature like this:

```rs
pub fn open<'a, C>(contract: C) -> Result<Self, KDBError<'a>>
    where
        C: Into<Option<Contract>>;
```

This way, we can call `KDB::open(None)` and `KDB::open(Contract { ... })` respectively, without having to specify `Some(Contract {})` in the second case, so it's a bit nicer than using `Option` directly.

I tried it here on [the Rust playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=ffe36ac84cd968901ac71bbc9502ca8c).